### PR TITLE
Fixes #20230 - Add time ended to error tasks widget

### PR DESCRIPTION
**Pull request by tbrisker**
Also made the table fixed with tooltips on overflow.

### CODE REVIEW

File: app/views/foreman_tasks/tasks/dashboard/_latest_tasks_in_error_warning.html.erb [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
-<table class="table table-striped">
+<table class="table table-striped table-fixed">
   <tr>
-    <th><%= _("Name") %></th>
-    <th><%= _("State") %></th>
-    <th><%= _("Result") %></th>
+    <th class="col-md-5"><%= _("Name") %></th>
+    <th class="col-md-2"><%= _("State") %></th>
+    <th class="col-md-2"><%= _("Result") %></th>
+    <th class="col-md-3"><%= _("Ended at") %></th>
```
Comment (adamruzicka) [inline app/views/foreman_tasks/tasks/dashboard/_latest_tasks_in_error_warning.html.erb]: Would it make sense to drop the 'at' from the header or reword it some other way? `Ended at: 5 minutes ago` feels weird, but maybe it's just me.

Comment (iNecas) [inline app/views/foreman_tasks/tasks/dashboard/_latest_tasks_in_error_warning.html.erb]: Just `Ended` should work fine, or?

Comment (tbrisker) [inline app/views/foreman_tasks/tasks/dashboard/_latest_tasks_in_error_warning.html.erb]: changed to Ended

**Comment by iNecas** [comment]
Thanks @tbrisker

**Comment by iNecas** [comment]
and @adamruzicka for review

**Comment by beav** [comment]
(sorry for the post-merge review, I didn't catch the issue til just now 🎣 )

If a task is in paused state, it won't have an ended_at and the widget will error. I had to use this line instead:

`<td><%= task.ended_at ? _('%s ago') % time_ago_in_words(task.ended_at) : '' %></td>`

Other than that, the change looks good.

**Comment by tbrisker** [comment]
@beav thanks for the catch, opened https://github.com/theforeman/foreman-tasks/pull/266

**Pull request merged**